src/components/fields/password-field/password-field.story.js

```diff
@@ -54,7 +54,7 @@ storiesOf('Components|Fields', module)
               isRequired={boolean('isRequired', false)}
               touched={boolean('touched', false)}
               name={text('name', '')}
-              value={text('value', value)}
```

Comment:
this was annoying, it's better to use the value so that the user can type into the field itself and not into the knob.